### PR TITLE
FEATURE: BEND-61 Get current situation from user

### PR DESCRIPTION
**Pull request by davidcode10**
Endpoint to get current situation from user

### CODE REVIEW

**Comment by whitehatdevv** [comment]
Pls @davidcode10 put a description here. All the PR must contain one

**Review by PatyVilas — APPROVED** [review]

File: src/routes/User.ts [hunk 1]
```diff
@@ -775,4 +775,26 @@ export const UserRoutes = (app: express.Application) => {
...
+  app.get(
+    UserEndpoints.CurrentSituation,
+    [
+      JWTMiddleware._verify,
+      AccessControlmiddleware._grantAccess((query, role) => {
+        switch (role) {
+          case PermissionRoles.USER:
+            return query.deleteOwn(_resource);
+          case PermissionRoles.ADMIN:
+            return query.deleteAny(_resource);
```
Comment (OvidioSantoro) [inline src/routes/User.ts:794]: The permissions we would need here are `query.readOwn` and `query.readAny` as this is a GET request (i.e.: read)

Comment (davidcode10) [reply src/routes/User.ts]: @OvidioSantoro thanks! I changed the permissions

**Review by Kehomaxd — APPROVED** [review]

**Review by OvidioSantoro — DISMISSED** [review]

File: src/routes/User.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+import express, {query} from 'express';
```
[the remark this answers is not in the document]
Comment (octapf) [reply src/routes/User.ts:1]: i think when sometimes we type query it may import it accidently from express

**Review by octapf — DISMISSED** [review]

**Review by OvidioSantoro — APPROVED** [review]

**Review by Kehomaxd — APPROVED** [review]

**Review by PatyVilas — APPROVED** [review]

**Review by octapf — APPROVED** [review]

**Pull request merged**